.openpublishing.redirection.json

```diff
@@ -50936,14 +50936,144 @@
         "redirect_document_id": false
     },
     {
-      "source_path": "docs-ref-services/preview/appinsights.md",
-      "redirect_url": "/azure/azure-monitor/app/java-in-process-agent",
-      "redirect_document_id": false
+        "source_path": "docs-ref-services/preview/appinsights.md",
+        "redirect_url": "/azure/azure-monitor/app/java-in-process-agent",
+        "redirect_document_id": false
     },
     {
-      "source_path": "docs-ref-services/latest/batchai.md ",
-      "redirect_url": "https://aka.ms/batchai-retirement",
-      "redirect_document_id": true
+        "source_path": "docs-ref-services/latest/batchai.md ",
+        "redirect_url": "https://aka.ms/batchai-retirement",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/batch-ai.md ",
+        "redirect_url": "https://aka.ms/batchai-retirement",
+        "redirect_document_id": true
+    },    
+    {
+        "source_path": "docs-ref-services/latest/appservice.md",
+        "redirect_url": "/java/api/overview/azure/app-service?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/batchai.md",
```

Comment:
I just happened to notice that [batchai.md redirects to an aka.ms link](https://github.com/Azure/azure-docs-sdk-java/pull/2189/files#diff-3fa6f2a8672a06f5c881c8621af9e49f935e65eb99e06ec73d42afe0b1323d7aL50944): 

```
      "source_path": "docs-ref-services/latest/batchai.md ",
      "redirect_url": "https://aka.ms/batchai-retirement",
      "redirect_document_id": true
``` 

I wonder which redirect configuration wins. I didn't check for that in my script :P 


Maybe we should exclude these or even redirect our `batch-ai.md` files to the aka.ms link.